test/tester.go

```diff
@@ -643,3 +665,44 @@ func (at *AccountsTester) UploadInfo(sl string) ([]api.UploadInfo, int, error) {
 	}
 	return resp, r.StatusCode, nil
 }
+
+/*** Stripe helpers ***/
+
+// StripeBillingPOST performs a `POST /stripe/billing`
+func (at *AccountsTester) StripeBillingPOST() (http.Header, int, error) {
+	r, err := at.Request(http.MethodPost, "/stripe/billing", nil, nil, nil, nil)
+	// We ignore the temporary redirect error because it's the expected
+	// behaviour of this endpoint.
+	if err != nil && !strings.Contains(err.Error(), "307 Temporary Redirect") {
+		return nil, r.StatusCode, err
+	}
+	return r.Header, r.StatusCode, nil
+}
+
+// StripeCheckoutPOST performs a `POST /stripe/checkout`
+func (at *AccountsTester) StripeCheckoutPOST(price string) (string, int, error) {
+	body := struct {
+		Price string
+	}{
+		price,
+	}
+	bodyBytes, err := json.Marshal(body)
+	if err != nil {
+		return "", http.StatusBadRequest, errors.AddContext(err, "failed to serialize request body")
+	}
+	resp := struct {
+		SessionID string
+	}{}
+	r, err := at.Request(http.MethodPost, "/stripe/checkout", nil, bodyBytes, nil, &resp)
+	return resp.SessionID, r.StatusCode, err
+}
+
+// StripePricesGET performs a `GET /stripe/prices`
+func (at *AccountsTester) StripePricesGET() ([]api.StripePrice, int, error) {
```

Comment:
Stripe `list` endpoints usually implement paging with `starting_after` and `ending_before`. They have a `limit` parameter but I think the default is like 10 or 20 and pretty sure it's capped to 100.

Reply:
We have three prices in total. Do we need to add pagination at this point?

Reply:
would at least set the limit to 100, easy win and if we add 8 prices we're not screwed